cinema/serializers.py

```diff
@@ -1,13 +1,83 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=100, required=True)
```

Comment:
You use both variants: required=True and sometimes without required statement. I think you should choose what you want to use